doc/api_assets/style.css

```diff
@@ -93,7 +93,10 @@ em code {
   color: white !important;
   margin: 0 0 1em 0;
   font-family: "Lato", "Lucida Grande", "Lucida Sans Unicode", "Lucida Sans", Verdana, Tahoma, sans-serif;
-  font-weight: 700;
+  white-space: pre-wrap;
+  word-wrap: break-word;
+  padding: 1em;
+  line-height: 1.5;
```

Comment:
I think I like it more with the padding. Another option would be to make it a `<p>` which already sets 1.5em line-height, so you save on declaration.